.gitignore

```diff
@@ -5,10 +5,16 @@
 .venv
 venv
 
+# Pyenv local overrides
+.python-version
+
 # Python cache files
 __pycache__/
 *.py[cod]
 
 # Build directories
 build
 
+# Ignore dev requirements files.
+images/airflow/2.*/requirements-dev.txt
```

Comment:
Once these are ignored from git, when a new image folder is created, e.g, 2.11, the line `open(requirements_path.resolve(), 'r')` in `generate_requirements` will throw error after cloning repo since file will not exist.

Reply:
Ah nevermind, this is just ignoring dev